Task7/task7.html

```diff
@@ -0,0 +1,164 @@
+                </div>
+                  
+            </div>
+        </div>
+
+    </div>
+    
+    <div class="div1">
+        <img id="imgLogo" class="img7" src ="imges/7.png"/>
+        <img id="theImage" class="img9" src ="imges/9.png"/> 
+        <div class="container" onclick="myFunction(this)">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>  
+    </div> 
+    <div class="div3">
+    <div class="divHelloTxt">
+            <span>Hello,</span><br>
+            Donec commodo leo est quis eleifend urna pellentesque eget. Proin scelerisque odio en nisi viverra, nec ullamcorper augue sagittis. Proin faucibus lorem nisl,
+    </div>   
+    </div>
+    
+    
+    <div class="div2">
+        <img src="imges/3.png" class="img3">
+        <img src="imges/4.png" class="img4">
+        <img src="imges/5.png" class="img5">
+        <img src="imges/6.png" class="img6">
+        <div class="readMore">
+            <a href="#">ــــــــ   read more</a>
+        </div>
+        
+        <div class="row"> 
+            <img class="imge" src ="imges/a.png"/>
+            <img class="imge" src ="imges/b.png"/><br>
+            <img class="imge" src ="imges/c.jpg"/>
+            <img class="imge" src ="imges/d.png"/>
+        </div>
+
+        <div class="div4">
+            
+            <span style="letter-spacing: 5px;font-weight: bold;">LOADING </span><br>
+            
+            <img class="img8" src ="imges/8.png"/>
+            <h1>
+                1.800.123.456
+            </h1>
+
+            <h6 style="color: #adadad;">
+                Donec commodo leo est quis eleifend urna pellentesque eget. 
+                Proin scelerisque odio eu nisi viverra, nec ullamcorper augue sagittis. 
+                Proin faucibus lor  or <a class="link2" href="#">contactme@mysite.com</a><br><br>
+            </h6>  
+            <div class="row1">
+                <a class="bottomLink" href="#">FACEBOOK</a>
+                <a class="bottomLink centerLink" href="#">TWITTER</a>
+                <a class="bottomLink" href="#">DRIBBBLE</a>       
+            </div>
+
+        </div> 
+
+        <div class="vl"></div>
+    </div>   
+
+    <script>
+        document.getElementById("myDIV").style.display = "none";
+        document.getElementById("myDIV2").style.display = "none";
+        
+        function myFunction(x) {
+            x.classList.toggle("change");
+            var y = document.getElementById("myDIV");
+            if (y.style.display === "none") {
+                document.getElementById("imgLogo").style.display = "none";
+                document.getElementById("theImage").style.filter="invert(100%)";
+                y.style.display = "block";
+                document.getElementById("myDIV2").style.display = "block";
+            } else {
+                y.style.display = "none";
+                document.getElementById("myDIV2").style.display = "none";
```

Comment:
myDIV2 rename this id

Reply:
https://github.com/RahafSalah/FrontEndTraining/pull/6